models/content.rb

```diff
@@ -11,6 +11,15 @@ class Content
   index({_type: 1, course_id: 1, pinned: -1, created_at: -1 }, {background: true} )
   index({_type: 1, course_id: 1, pinned: -1, comment_count: -1, created_at: -1}, {background: true})
   index({_type: 1, course_id: 1, pinned: -1, "votes.point" => -1, created_at: -1}, {background: true})
+  index({_type: 1, course_id: 1, pinned: -1, last_activity_at: -1, created_at: -1}, {background: true})
```

Comment:
I thought the order was 
# _type
# course_id
# pinned
# created_at
# last_activity_at

?